lib/datadog/tracing/transport/trace_formatter.rb

```diff
@@ -54,6 +60,11 @@ def format!
           tag_sampling_priority!
           tag_profiling_enabled!
 
+          return trace unless first_span
+
+          tag_git_repository_url!
+          tag_git_commit_sha!
+
           trace
```

Comment:
Minor: It's very much stylistic thing, but at least for me this seems to read a bit better?

```suggestion
          if first_span
            tag_git_repository_url!
            tag_git_commit_sha!
          end

          trace
```

Also -- I'm not quite sure I understand, when would a trace not have any spans? (e.g. `trace.spans.first == nil`)